Salience/Salience.ino

```diff
@@ -0,0 +1,85 @@
+#include <Encoder.h>
```

Comment:
The directory for this file should be `Salience_bias_box/src`

Reply:
got it, will change that